.github/workflows/sync-docs.yml

```
@@ -15,11 +25,37 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set commit variables
+        id: commits
+        run: |
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+            # Manual run
+            BEFORE_COMMIT="${{ github.event.inputs.before_commit }}"
+            AFTER_COMMIT="${{ github.event.inputs.after_commit }}"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)